bitnami/contour/values.yaml

```diff
@@ -0,0 +1,234 @@
+## Default values for contour.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry, imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+
+## To configure Contour, you must specify ONE of the following two
+## options.
+#
+## existingConfigMap specifies the name of an externally-defined
+## ConfigMap to use as the configuration. Helm will not manage the
+## contents of this ConfigMap, it is your responsibility to create it.
+#
+# existingConfigMap: contour
+#
+## configInline specifies Contour's configuration directly, in yaml
+## format. When configInline is used, Helm manages Contour's
+## configuration ConfigMap as part of the release, and
+## existingConfigMap is ignored.
+## Refer to https://projectcontour.io/docs/v1.2.1/configuration/ for
+## available options.
+#
+configInline:
+  # should contour expect to be running inside a k8s cluster
+  # incluster: true
+  #
+  # path to kubeconfig (if not running inside a k8s cluster)
+  # kubeconfig: /path/to/.kube/config
+  #
+  # Client request timeout to be passed to Envoy
+  # as the connection manager request_timeout.
+  # Defaults to 0, which Envoy interprets as disabled.
+  # Note that this is the timeout for the whole request,
+  # not an idle timeout.
+  # request-timeout: 0s
+  # disable ingressroute permitInsecure field
+  disablePermitInsecure: false
+  tls:
+  #   minimum TLS version that Contour will negotiate
+  #   minimum-protocol-version: "1.1"
+  # The following config shows the defaults for the leader election.
+  ## This needs to be edited by when you deploy to a namespace other than projectcontour
+  # leaderelection:
+  #   configmap-name: leader-elect
+  #   configmap-namespace: projectcontour
```

Comment:
Do you have an advice on how to handle the namespace dependant on the deployed namespace properly.
I am currently thinking of tplValue from other charts.